### PR TITLE
bibauthorid: preparing for newlabs switch

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

**Comment by kaplun** [comment]
@tsgit I will need to deploy this tomorrow in order to adapt to the new URLs on Labs/Invenio 3. Let me know if you see anything obviously wrong :)

**Comment by kaplun** [comment]
See also inspirehep/inspire#283

File: modules/bibauthorid/etc/templates/personal_details_box.html [hunk 1]
```diff
@@ -143,7 +143,7 @@ <h4 class="list-group-item-heading">Create One</h4>
 		{% if link_to_record %}
 		<a href="{{ hepnames_link }}{{ record.record_id }}"type="button" class="btn btn-danger">HepNames Record</a>
 		{% endif %}
-		<a href="{{ update_link }}{{ record.record_id }}"type="button" class="btn btn-default">Update Details</a>
+		<a href="{{ update_link_prefix }}{{ record.record_id }}{{ update_link_suffix }}"type="button" class="btn btn-default">Update Details</a>
```
Comment (tsgit) [inline modules/bibauthorid/etc/templates/personal_details_box.html]: needs a space before `type="button"` ?

**Review by tsgit — APPROVED** [review]
👍 LGTM
would be better to not hardcode the urls but ok for quick tweak

**Pull request merged**